src/live.asp.net/Views/Shared/_YouTubeShow.cshtml

```diff
@@ -20,7 +20,7 @@
                     <span class="show-date">@Model.ShowDate.DateTime.ToString("MMM dd, yyyy")</span>
                     @if (Model.IsNew)
                     {
-                        <span class="label label-success">New</span>
+                        <span id="newEpisode" class="label label-success">New</span>
```

Comment:
You can't use an `id` attribute here as this is repeated and `id` must be unique. Just create a new class named `label-new-show` and style it via that. Also, can you include a screenshot of the before and after effects of the change please.